sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultDataStore.java

```diff
@@ -687,7 +687,7 @@ public void beforeWrite(Collection<RecordUpdate> updateBatch) {
                     }
                 }
                 if (!updateRefs.isEmpty()) {
-                    _eventBus.post(new UpdateIntentEvent(this, updateRefs));
+                    _eventWriterRegistry.getDatabusWriter().writeEvent(new UpdateIntentEvent(this, updateRefs));
```

Comment:
This is pretty minor, but can we remove `UpdateIntentEvent` completely and just pass the list of `UpdateRef`s?  It isn't buying us anything.